ale/drivers/mro_drivers.py

```diff
@@ -147,4 +147,16 @@ def spacecraft_name(self):
         name_lookup = {
             'MARS_RECONNAISSANCE_ORBITER': 'MRO'
         }
-        return name_lookup[self.label['SPACECRAFT_NAME']]
+        return name_lookup[super().spacecraft_name]
+
+    @property
+    def detector_start_line(self):
+        return 0
+
+    @property
+    def detector_start_sample(self):
+        return self.label.get('SAMPLE_FIRST_PIXEL', 0)
+    
+    @property
+    def sensor_model_version(self):
+        return 1
```

Comment:
From the ISIS CTX plugin file.  Thanks @paarongiroux for describing where you got this for dawn!

Reply:
From our discussion: If this is not set, the isis code as a built in default of 1 (one).

Reply:
That code needs to be updated though, so this should probably stay and then we can get the Base updated in another PR.